redisson-hibernate/redisson-hibernate-4/src/main/java/org/redisson/hibernate/region/BaseRegion.java

```diff
@@ -39,7 +39,7 @@
  */
 public class BaseRegion implements TransactionalDataRegion, GeneralDataRegion {
 
-    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private static final Logger logger = LoggerFactory.getLogger(BaseRegion.class);
```

Comment:
this needs to remain unchanged or be added to all inheritances with concrete class, not BaseRegion.